app/src/main/java/org/moire/opensudoku/gui/inputmethod/IMControlPanel.java

```diff
@@ -45,7 +45,6 @@
 	public static final int INPUT_METHOD_SINGLE_NUMBER = 1;
 	public static final int INPUT_METHOD_NUMPAD = 2;
 
-	private Context mContext;
```

Comment:
LinearLayout being a subclass of ViewGroup (which is a subclass of View) already has a reference to a Context object.  The class extends LinearLayout thus the extra reference to a Context object is redundant.